lib/apus/adapters/local_adapter.ex

```diff
@@ -1,13 +1,18 @@
 defmodule Apus.LocalAdapter do
   @moduledoc """
+  Deliver messages to a local inbox, available to view if
+  `Apus.SentMessagesViewerPlug` is installed in your router.
   """
 
   @behaviour Apus.Adapter
 
   alias Apus.SentMessages
 
   def deliver(message, _config) do
-    SentMessages.push(message)
+    case SentMessages.push(message) do
+      :ok -> {:ok, message}
+      error -> {:error, error}
+    end
```

Comment:
The `SentMessages.push` function always returns the same result so I don't think we need the `case` here.

Can we remove the `case` and update `SentMessages.push` to return `{:ok, message}`?